### PR TITLE
docs: Update README

### PR DESCRIPTION
**Pull request by danielpacak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,33 +4,84 @@
...
+- [Deployment](#deployment)
 - [Configuration](#configuration)
 - [Documentation](#documentation)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Getting started
+
+These instructions will get you a copy of the adapter service up and running on your local machine for development
+and testing purposes. See [deployment](#deployment) for notes on how to deploy on a live system.
```
Comment (lizrice) [inline README.md]: Strictly I think the "Getting Started" instructions just say how to build and run the unit and integration tests. 

Do you maybe want to leave the minikube instructions or is that no longer useful?

File: README.md [hunk 2]
```diff
@@ -4,33 +4,84 @@
...
+make test-integration
+```
+
+## Deployment
+
+### Kubernetes
+
+1. Create Kubernetes deployment and service for the scanner adapter:
+   ```
+   $ kubectl apply -f kube/harbor-scanner-trivy.yaml
```
Comment (lizrice) [inline README.md]: Does this assume a locally-built image or will it pull from a registry somewhere?

File: README.md [hunk 3]
```diff
@@ -54,6 +105,15 @@ curl -v http://localhost:8080/api/v1/metadata
 - [Architecture](./docs/ARCHITECTURE.md): architectural decisions behind designing harbor-scanner-trivy.
 - [Releases](./docs/RELEASES.md): how to release a new version of harbor-scanner-trivy.
 
+## Contributing
+
+Please read [CONTRIBUTING.md](CONTRIBUTING.md) for details on our code of conduct, and the process for submitting pull
```
Comment (lizrice) [inline README.md]: We do have a standard COC (see https://github.com/aquasecurity/.github/blob/master/CODE_OF_CONDUCT.md - the .github files get applied to all projects by default)

File: README.md (hunk 1 above)
Comment (danielpacak) [reply README.md]: You're right. I've reverted instructions for running on minikube.

File: README.md (hunk 2 above)
Comment (danielpacak) [reply README.md]: I've clarified that by default it's the latest image published to Docker Hub. For local deployment you have to change it manually with `kubectl set image deployment ...`.

At some point we should have a helm chart with image variable and some default value.

File: README.md (hunk 3 above)
Comment (danielpacak) [reply README.md]: Update the link.

**Pull request merged**